docs/contributing/documentation-style-guide.md

```diff
@@ -201,11 +216,11 @@ Styles](https://medium.com/better-programming/string-case-styles-camel-pascal-sn
 
 **Examples**:
 
->`hook-and-plugins.md`,
+>`hook-and-plugins.md`
```

Comment:
these quote blocks were added to the other two MD titles to continue the formatting aesthetics.